hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis_snapshot/OmRatisSnapshotProvider.java

```diff
@@ -22,6 +22,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.FileOutputStream;
 import java.net.HttpURLConnection;
```

Comment:
Please don't merge with outdated CI results.  Last run, although triggered on Feb 21, was against code from Feb 13, because that's when the last commit was pushed to the PR:

```
  HEAD is now at 7062329 Merge 3cb99a2b4539b3f57a1da40118f8673aed7552f3 into 28b9a03204e2238ca9b01f403811f433685a4440
```

https://github.com/apache/ozone/actions/runs/13305097960/job/37592785787#step:2:90

```
commit 28b9a03204e2238ca9b01f403811f433685a4440
Author:     Takanobu Asanuma <tasanuma@apache.org>
AuthorDate: Thu Feb 13 16:28:01 2025 +0900
Commit:     GitHub <noreply@github.com>
CommitDate: Thu Feb 13 08:28:01 2025 +0100

    HDDS-12312. NodeManager log aggregation to Ozone FileSystem fails. (#7856)
```

Reply:
Sure , will keep this in mind. 
After how many days/commits b/w PR branch & master should one rebase and re-run? Ideally it should be after every new commit in master as any new commit can cause the CI to fail but do we actually follow this i.e re-running CI even if no conflicts reported in the PR

Reply:
I guess it depends on the kind of changes done on `master` in the meantime.  With new checkstyle rules it can get broken in one commit.  Sorry I didn't notice that this PR would fail despite having no conflicts.

If in doubt, check out the PR locally, merge `master` into it, then run some quick sanity checks:

```bash
gh pr checkout 7861
git merge --no-commit origin/master
mvn -DskipTests -DskipRecon -DskipShade clean verify
./hadoop-ozone/dev-support/checks/checkstyle.sh
./hadoop-ozone/dev-support/checks/pmd.sh
```